assets/js/index.js

```diff
@@ -0,0 +1,900 @@
+
+twentytwenty.intrinsicRatioVideos = {
+
+	init: function () {
+		this.makeFit();
+
+		window.addEventListener('fit-videos', function () {
+			this.makeFit();
+		});
+
+		window.addEventListener('resize', function () {
+			this.makeFit();
+		});
+	},
+
+	makeFit: function () {
+		document.querySelectorAll('iframe, object, video').forEach(function (video) {
+			var container = video.parentNode;
+
+			// Skip videos we want to ignore
+			if (video.classList.contains('intrinsic-ignore') || video.parentNode.classList.contains('intrinsic-ignore')) {
+				return true;
+			}
+
+			if (!video.dataset.origwidth) {
+				// Get the video element proportions
+				video.setAttribute('data-origwidth', video.width);
+				video.setAttribute('data-origheight', video.height);
+			}
+
+			var iTargetWidth = container.offsetWidth;
+
+			// Get ratio from proportions
+			var ratio = iTargetWidth / video.dataset.origwidth;
+
+			// Scale based on ratio, thus retaining proportions
+			video.style.width = iTargetWidth + 'px';
+			video.style.height = video.dataset.origheight * ratio + 'px';
+		});
+	},
+
+}; // twentytwenty.instrinsicRatioVideos
+
+/*	-----------------------------------------------------------------------------------------------
+	Smooth Scroll
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.smoothScroll = {
+
+	init: function () {
+		// Scroll to anchor
+		this.scrollToAnchor();
+
+		// Scroll to element
+		this.scrollToElement();
+	},
+
+	// Scroll to anchor
+	scrollToAnchor: function () {
+		var anchorElements = document.querySelectorAll('a[href*="#"]');
+		var anchorElementsList = Array.prototype.slice.call(anchorElements);
+		anchorElementsList.filter(function (element) {
+			if (element.href === '#' || element.href === '#0' || element.classList.contains('.do-not-scroll') || element.classList.contains('skip-link')) {
+				return false;
+			}
+			return true;
+		}).forEach(function (element) {
+			element.addEventListener('click', function (event) {
+				// On-page links
+				if (window.location.hostname === event.target.hostname) {
+					// Figure out element to scroll to
+					var target = window.location.hash !== '' && document.querySelector(window.location.hash);
+					target = target ? target : event.target.hash !== '' && document.querySelector('#' + event.target.hash);
+
+					// Does a scroll target exist?
+					if (target) {
+						// Only prevent default if animation is actually gonna happen
+						event.preventDefault();
+
+						// Get options
+						var additionalOffset = event.target.dataset.additionalOffset;
+						var scrollSpeed = event.target.dataset.scrollSpeed ? event.target.dataset.scrollSpeed : 500;
+
+						// Determine offset
+						var originalOffset = target.getBoundingClientRect().top + window.pageYOffset;
+						var scrollOffset = additionalOffset ? originalOffset + additionalOffset : originalOffset;
+
```

Comment:
You could add this in case the adminbar is active:
```
var adminBar = document.querySelector( '#wpadminbar' );

if( adminBar && event.target.className === 'to-the-top' ) {
    scrollOffset = scrollOffset - adminBar.getBoundingClientRect().height;
}
```